src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -20,7 +20,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="System\Threading\Tasks\TaskExtensions.CoreCLR.cs" />
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
-      <OSGroup>Windows_NT</OSGroup>
+      <OSGroup>$(TestOsTarget)</OSGroup>
```

Comment:
I'll change System.Diagnostics.Debug.csproj's default Configuration to be $(TargetOS)_Debug instead of Windows_Debug to mitigate projects that are targeting AnyOS referencing it and getting the Windows binary